core/templates/pages/contributor-dashboard-page/services/contribution-and-review.service.ts

```diff
@@ -165,35 +168,37 @@ export class ContributionAndReviewService {
   }
 
   async getUserCreatedQuestionSuggestionsAsync(
-      shouldResetOffset: boolean = true,
-      topicName: string = 'All'):
+      shouldResetOffset: boolean = true):
   Promise<FetchSuggestionsResponse> {
     return this.fetchSuggestionsAsync(
-      this.userCreatedQuestionFetcher, shouldResetOffset, topicName);
+      this.userCreatedQuestionFetcher,
+      shouldResetOffset);
   }
 
   async getReviewableQuestionSuggestionsAsync(
-      shouldResetOffset: boolean = true,
-      topicName: string = 'All'):
+      shouldResetOffset: boolean = true):
   Promise<FetchSuggestionsResponse> {
     return this.fetchSuggestionsAsync(
-      this.reviewableQuestionFetcher, shouldResetOffset, topicName);
+      this.reviewableQuestionFetcher,
+      shouldResetOffset);
   }
 
   async getUserCreatedTranslationSuggestionsAsync(
-      shouldResetOffset: boolean = true,
-      topicName: string = 'All'):
+      shouldResetOffset: boolean = true):
   Promise<FetchSuggestionsResponse> {
     return this.fetchSuggestionsAsync(
-      this.userCreatedTranslationFetcher, shouldResetOffset, topicName);
+      this.userCreatedTranslationFetcher,
+      shouldResetOffset);
   }
 
   async getReviewableTranslationSuggestionsAsync(
       shouldResetOffset: boolean = true,
-      topicName: string = 'All'):
+      explorationId: string):
```

Comment:
You can add newline after "string" snd them have `): Promise<FetchSuggestionsResponse> {` on the next line for better readability.

Reply:
Ditto above

Reply:
Done.